src/mono/msbuild/apple/build/AppleApp.targets

```diff
@@ -36,7 +36,7 @@
 
     <!-- common linker arguments for app and library builds -->
     <ItemGroup>
-      <_CommonLinkerArgs Condition="'$(TargetOS)' != 'tvos'" Include="-framework GSS" />
```

Comment:
Just a question: 
Should the condition also include `and '$(TargetOS)' != 'tvossimulator'`?

Reply:
Good catch. Yeah, it should.